### DISCUSSION TITLE
How to debug SSG error? In my case, it build successfully with `ssg.strict: true`, but actually failed

### DISCUSSION THREAD

**Discussion by JounQin** [web-infra-dev/rspress#1659] [Q&A]
<img width="613" alt="image" src="https://github.com/user-attachments/assets/a0ad54b6-6b33-4001-8a44-a6c62c33bec3" />

https://react.dev/errors/419?invariant=419

```log
The server could not finish this Suspense boundary, likely due to an error during server rendering. Switched to client rendering.
```

**Comment by SoonIter** [comment]
it may be related to `Suspense`,  any reproduction?

**Reply by SoonIter** [reply]
@JounQin hello, how did you solve this problem? 😂 I encountered the same problem.

**Reply by JounQin** [reply]
I didn't resolve it... it just fallbacks to client side renderring. 😂
